src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -2,83 +2,33 @@ import React, { useEffect, useState } from 'react';
 
 import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
 import {
-  Button,
-  DescriptionList,
-  DescriptionListDescription,
-  DescriptionListGroup,
-  DescriptionListTerm,
-  List,
-  ListItem,
-  Popover,
-  Spinner,
-  Tab,
-  Tabs,
-  TabTitleText,
+  ExpandableSection,
   Text,
   TextContent,
-  TextList,
-  TextListItem,
-  TextListItemVariants,
-  TextListVariants,
   TextVariants,
 } from '@patternfly/react-core';
-import { HelpIcon } from '@patternfly/react-icons';
-import {
-  TableComposable,
-  Tbody,
-  Td,
-  Th,
-  Thead,
-  Tr,
-} from '@patternfly/react-table';
-import PropTypes from 'prop-types';
 
-import ActivationKeyInformation from './ActivationKeyInformation';
+import {
+  ContentList,
+  FSCList,
+  ImageDetailsList,
+  ImageOutputList,
+  RegisterLaterList,
+  RegisterNowList,
+  TargetEnvAWSList,
+  TargetEnvAzureList,
+  TargetEnvGCPList,
+  TargetEnvOtherList,
+} from './ReviewStepTextLists';
 
-import { RELEASES, UNIT_GIB, UNIT_MIB } from '../../../constants';
+import { UNIT_GIB } from '../../../constants';
 import { useGetAWSSourcesQuery } from '../../../store/apiSlice';
 import isRhel from '../../../Utilities/isRhel';
 import { googleAccType } from '../steps/googleCloud';
 
-const FSReviewTable = ({ ...props }) => {
-  return (
-    <TableComposable
-      aria-label="File system configuration table"
-      variant="compact"
-    >
-      <Thead>
-        <Tr>
-          <Th>Mount point</Th>
-          <Th>File system type</Th>
-          <Th>Minimum size</Th>
-        </Tr>
-      </Thead>
-      <Tbody data-testid="file-system-configuration-tbody-review">
-        {props.fsc.map((r, ri) => (
-          <Tr key={ri}>
-            <Td className="pf-m-width-30">{r.mountpoint}</Td>
-            <Td className="pf-m-width-30">xfs</Td>
-            <Td className="pf-m-width-30">
-              {r.size}{' '}
-              {r.unit === UNIT_GIB
-                ? 'GiB'
-                : r.unit === UNIT_MIB
-                ? 'MiB'
-                : 'KiB'}
-            </Td>
-          </Tr>
-        ))}
-      </Tbody>
-    </TableComposable>
-  );
-};
-
-FSReviewTable.propTypes = {
-  fsc: PropTypes.arrayOf(PropTypes.object).isRequired,
-};
-
 const ReviewStep = () => {
-  const [activeTabKey, setActiveTabKey] = useState(0);
+  const [isExpanded, setIsExpanded] = useState(false);
+
   const [minSize, setMinSize] = useState();
```

Comment:
It seems like minSize is only used by the <FSCList minSize={minSize} /> component. Would it make sense to push this state 'down' into the <FSCList /> component instead of managing it here in the parent?

Reply:
Lines 48-62 in the useEffect hook are also related to this and could be handled in the `<FSCList />` component, I think.

Reply:
It makes perfect sense to move those, done!